src/os/kstore/KStore.cc

```diff
@@ -3169,6 +3179,45 @@ int KStore::_clone_range(TransContext *txc,
   return r;
 }
 
+int KStore::_move_ranges_destroy_src(TransContext *txc,
+                           CollectionRef& c,
+                           OnodeRef& srco,
+                           CollectionRef& basec,
+                           OnodeRef& baseo,
+                           vector<boost::tuple<uint64_t, uint64_t, uint64_t>> move_info)
+{
+  int r = 0;
+  bufferlist bl;
+  baseo->exists = true;
+  _assign_nid(txc, baseo);
+
+//for loop
```

Comment:
I'm pretty sure most people will recognize that it's a for loop without the comment.